### DISCUSSION TITLE
QA Review

### DISCUSSION THREAD

**Discussion by alboss** [alboss/ui-team-1#3] [General]
Quality Assurance (QA) review on the other team's website (assigned in the table above) to evaluate basic navigation functionality, spelling, grammar, browser compatibility, and accessibility.

Attach a report as described in the assignment.

**Comment by MaxJoe99** [comment]
[QAA Checklist-1.docx](https://github.com/alboss/ui-team-1/files/10917028/QAA.Checklist-1.docx)

**Comment by AndrewFales** [comment]
[QAA Checklist-Andrew.docx](https://github.com/alboss/ui-team-1/files/10917960/QAA.Checklist-Andrew.docx)

**Comment by juliannalin** [comment]
[QAA Checklist-Julianna.docx](https://github.com/alboss/ui-team-1/files/10918124/QAA.Checklist-Julianna.docx)

**Comment by C-Richter** [comment]
[Richter QAA Checklist.docx](https://github.com/alboss/ui-team-1/files/10965511/Richter.QAA.Checklist.docx)

**Comment by sofiaostash** [comment]
[QAA Checklist 2.docx](https://github.com/alboss/ui-team-1/files/10978030/QAA.Checklist.2.docx)
